src/components/pages/Room/Messages.tsx

```diff
@@ -59,24 +52,15 @@ const Message = ({ message, author, isOwned }: { message: IMessage, author: IAcc
     const rootClasses = cx(classes.message, { [classes.ownedMessage]: isOwned, [classes.notOwnedMessage]: !isOwned });
 
     useEffect(() => {
-        let elements = document.querySelectorAll(".messages-container > div");
-        let height = elements.forEach(message => {
-            //When we fix bug with roll text we know exacly how higher shuld be element with 4 lines then we can change px
-            if(message.clientHeight >= 150) {
-                message.className = classes.messageManyLinesRoot
-            }
-        })
-        return height;
-    })
-
-    // margin: isHeightMessage ? "8px": "15px"
-
-    return <Flex alignItems="flex-end" className={isOwned ? classes.ownedMessage : classes.messageRoot} style={{ alignSelf: isOwned ? "flex-end" : "flex-start"}}>
-    {!isOwned && author?.avatar_url && <img alt="author avatar" src={author.avatar_url} className={classes.avatar} />}
-    <div className={rootClasses}>
-        <ReactMarkdown source={value} />
-    </div>
-</Flex>
+        setHeight((ref as any).current.offsetHeight);
+    }, [])
+
+    return <Flex alignItems="flex-end" className={isOwned ? classes.ownedMessage : classes.messageRoot} style={{ alignSelf: isOwned ? "flex-end" : "flex-start", margin: height > 150 ? "8px":undefined }}>
```

Comment:
```suggestion
    return <Flex alignItems="flex-end" className={isOwned ? classes.ownedMessage : classes.messageRoot} style={{ alignSelf: isOwned ? "flex-end" : "flex-start", margin: height > 150 ? "8px" : 0 }}>
```
Undefined is actually not a valid CSS value. Change it to `0`, `"0px"` or just `"initial"`.

Reply:
I checked `0`, `"0px"` and `"initial"` all has these same result. Messages don't have margin only messages > 150px . 
Cascading rules (the most important are inline styles). I checked once again for `null` and I have seen I watched on message what has less 150px every message over 150px has correct margin with `null`.